app/src/main/java/hous/release/android/presentation/tutorial/TutorialViewModel.kt

```diff
@@ -20,13 +22,13 @@ class TutorialViewModel @Inject constructor(
 
     init {
         viewModelScope.launch {
-            _isTutorialState.value = tutorialDataSource.getShowTutorial()
+            _isTutorialState.value = getSkipTutorialUseCase.invoke()
```

Comment:
```suggestion
            _isTutorialState.value = getSkipTutorialUseCase()
```
operator fun invoke() 이기 때문에 이렇게 해도 작동이 됩니다

Reply:
> operator fun invoke() 이기 때문에 이렇게 해도 작동이 됩니다

이러면 isTutorialState를 nullable하게 바꿔야하네요..

Reply:
그건 usecase 때문이 아니라 suspend 함수가 return 값을 가지고 있기 때문이라고 생각이 드는데...
usecase로 바꾸기 전엔 잘 작동이 되었나요??

Reply:
> 그건 usecase 때문이 아니라 suspend 함수가 return 값을 가지고 있기 때문이라고 생각이 드는데... usecase로 바꾸기 전엔 잘 작동이 되었나요??

하하 잘 되네요 저 코드 지웠습니다 init만 usecase로 사용해요